store/tikv/gcworker/gc_worker.go

```diff
@@ -790,8 +790,11 @@ const (
 // needsGCOperationForStore checks if the store-level requests related to GC needs to be sent to the store. The store-level
 // requests includes UnsafeDestroyRange, PhysicalScanLock, etc.
 func needsGCOperationForStore(store *metapb.Store) (bool, error) {
-	engineLabel := ""
+	if store.State == metapb.StoreState_Tombstone {
```

Comment:
LGTM, better add some comments to make others aware of this condition which is easy to mistake.

Reply:
added